sxt/curve_g1/operation/add.h

```diff
@@ -17,21 +17,75 @@
 #pragma once
 
 #include "sxt/base/macro/cuda_callable.h"
+#include "sxt/curve_g1/operation/mul_by_3b.h"
+#include "sxt/curve_g1/type/element_p2.h"
+#include "sxt/field12/operation/add.h"
+#include "sxt/field12/operation/mul.h"
+#include "sxt/field12/operation/sub.h"
 
 namespace sxt::cg1t {
 struct element_affine;
-struct element_p2;
 } // namespace sxt::cg1t
 
 namespace sxt::cg1o {
+//--------------------------------------------------------------------------------------------------
+// add_inplace
+//--------------------------------------------------------------------------------------------------
+/*
+ p = p + q (q value is preserved but not const to satisfy concept requirements)
```

Comment:
> (q value is preserved but not const to satisfy concept requirements)

Why can't it be const and satisfy the concept?

Reply:
This was just my knowledge gap. I wasn't sure how to verify if it could be `const` without updating the `cbindings` to do MSM with `curve_g1` elements. 

I've updated it to `const`.